get_data/graphs_for_server.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+import pandas as pd
+import matplotlib
+from matplotlib import pyplot as plt
+
```

Comment:
Nice consistent use of naming conventions for functions and variables throughout 👍